src/Stryker.Core/Stryker.Core/Reporters/HtmlReporter/HtmlReporter.cs

```diff
@@ -38,22 +37,30 @@ public void OnAllMutantsTested(IReadOnlyProjectComponent reportComponent)
 
             WriteHtmlReport(reportPath, mutationReport.ToJsonHtmlSafe());
 
-            // to make path clickable it should always start with: file:///
-            var reportUri = reportPath.Replace("\\", "/");
-            reportUri = reportUri.StartsWith("/") ? reportUri : "/" + reportUri;
-            reportUri = "file://" + reportUri;
+            var reportUri = "file://" + reportPath.Replace("\\", "/");
 
             if (_options.ReportTypeToOpen == Options.Inputs.ReportType.Html)
             {
                 _processWrapper.Open(reportUri);
-            } else
+            }
+            else
+            {
+                _console.MarkupLine("[Cyan]Hint: by passing \"--open-report or -o\" the report will open automatically once Stryker is done.[/]");
+            }
+
+            _console.WriteLine();
+            _console.MarkupLine("[Green]Your html report has been generated at:[/]");
+
+            if (_console.Profile.Capabilities.Links)
```

Comment:
It appears the 'Ansi link shortcuts' capabilities check of Spectre.Console is not that advanced, it will probably only detect cmd.exe as not supporting Ansi links. I tested this on visual studio terminal, git bash, powershell, Gnome terminal and Guake. All showed as supporting Ansi links which is not technically true. However all do support Ansi so they simply ignored the special ansi link characters and only printed the link text. This is fine, however it does mean that we must print the full report path to disk. I had preferred to print something like [your report is here](#here) instead but at this time that does not seem possible.

See: https://github.com/spectreconsole/spectre.console/blob/52c1d9122bce4c73ae9205e2f2aff01cba03a4c4/src/Spectre.Console/AnsiConsoleFactory.cs#L49

Reply:
Created an issue at spectre.console: https://github.com/spectreconsole/spectre.console/issues/764